ui/src/app/job-list/job-list.component.spec.ts

```diff
@@ -98,7 +99,8 @@ describe('JobListComponent', () => {
       providers: [
         {provide: JobManagerService, useValue: fakeJobService},
         {provide: CapabilitiesService, useValue: new FakeCapabilitiesService(capabilities)},
-        JobListResolver
+        JobListResolver,
+        RouteReuse
```

Comment:
Can you add a test case that exercises the "stale" setting?

Reply:
Done.